src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/transport/action/addpolicy/TransportAddPolicyAction.kt

```diff
@@ -60,12 +81,36 @@ class TransportAddPolicyAction @Inject constructor(
         private val request: AddPolicyRequest
     ) {
         private lateinit var startTime: Instant
+        private val indicesToAdd = mutableMapOf<String, String>() // uuid: name
```

Comment:
We should take advantage of typealiases for this stuff since it's confusing just seeing string :P
But, can do that in later PR